staging/src/k8s.io/kubectl/pkg/proxy/proxy_server.go

```diff
@@ -61,6 +61,9 @@ type FilterServer struct {
 
 // MakeRegexpArray splits a comma separated list of regexps into an array of Regexp objects.
 func MakeRegexpArray(str string) ([]*regexp.Regexp, error) {
+	if str == "" {
```

Comment:
I would suggest:
```
if len(strings.TrimSpace(str)) == 0 {
```

This avoid also cases like:
```
 str := ""
 str := "    "
```

Reply:
It would be nice, but not a hard requirement, imo.